policyengine_canada/variables/gov/provinces/yt/tax/income/benefits/rebates/ygcpri/yt_government_carbon_price_rebate.py

```diff
@@ -0,0 +1,30 @@
+from policyengine_canada.model_api import *
+
+
+class yt_government_carbon_price_rebate(Variable):
+    value_type = float
+    entity = Household
+    label = "Yukon government carbon price rebate"
+    unit = CAD
+    definition_period = YEAR
+    reference = "https://www.canada.ca/en/revenue-agency/services/child-family-benefits/provincial-territorial-programs/yukon.html#YGCPRI"
+    defined_for = ProvinceCode.YT
+
+    def formula(household, period, parameters):
+        children = household("yt_ygcpri_eligible_children", period)
```

Comment:
Since this is just one extra line of code, we can skip the two extra variables for age comparisons and do it here.
```suggestion
        p = parameters(period).gov.provinces.yt.benefits.rebates.ygcpri
        person = household.members
        child = person("age", period) < p.child_ineligible_age
        children = household.sum(child)
```